test/k8sT/Services.go

```diff
@@ -1956,6 +2001,7 @@ var _ = Describe("K8sServicesTest", func() {
 		SkipItIf(helpers.DoesNotRunOnNetNextOr419Kernel, "Supports IPv4 fragments", func() {
 			DeployCiliumAndDNS(kubectl, ciliumFilename)
 			testIPv4FragmentSupport()
+			testIPv4FragmentMetrics()
```

Comment:
I wonder if we wouldn't benefit from merging `testIPv4FragmentSupport` and `testIPv4FragmentMetrics`. Right now, `testIPv4FragmentMetrics` checks fragmentation metrics for a single path, while `testIPv4FragmentSupport` already has tests for several different paths (via ClusterIP, with IPv4/IPv6, to remote and local nodes). If we merge the two functions, we could check metrics after each fragmented request and would avoid duplicated requests.